### DISCUSSION TITLE
How to send an vector of address

### DISCUSSION THREAD

**Discussion by memxor** [aptos-labs/aptos-developer-discussions#95] [Questions]
I've a game, and I want to send the addresses of the players as an array to the Smart Contract. How do I do that from the client? This is the parameter in the smart contract `winning_order: vector<address>`.

**Comment by 0xmaayan** [comment]
What client do you use?

If you use Aptos TS SDK, can do
```
const transaction = await aptos.transaction.build.simple({
    sender: sender.accountAddress,
    data: {
      function: "moduleAddress::moduleName::moduleFunction",
      typeArguments: [],
      functionArguments: ["0x123", "0x456"],
    },
  });

const committedTxn = await aptos.signAndSubmitTransaction({ signer: sender, transaction: transaction });
```

If you are using CLI, can do
```
aptos move run --function-id moduleAddress::moduleName::moduleFunction --args "address:[0xa1,0xb2,0xc3]"
```

**Reply by memxor** [reply]
I'm using Unity C# and Aptos UnitySDK, it used to be exactly like this before where I could `functionArguments: ["0x123", "0x456"],` but then it got changed. Now it uses `Sequence`. so if I had to send a u64 I can do `new U64(my_num)`, but there is nothing there for Arrays/vectors. 

@gregnazario would you have any idea?
